onnxruntime/python/tools/quantization/operators/reshape.py

```diff
@@ -0,0 +1,31 @@
+import onnx
```

Comment:
nice! could you add an unit test for it?

Reply:
Added!

Reply:
Thanks for your contribution! LGTM overall. Could you please remove the debugger break point in the unit test line 79?

Reply:
@yufenglee Fixed, sorry about that.